feconf.py

```diff
@@ -650,19 +650,26 @@ def get_empty_ratings():
 MAX_PLAYTHROUGHS_FOR_ISSUE = 5
 # Probability of recording a playthrough.
 RECORD_PLAYTHROUGH_PROBABILITY = 0.2
-# IDs of allowed types of issues.
-ALLOWED_ISSUE_IDS = [
+# Types of allowed issues.
+ALLOWED_ISSUE_TYPES = [
     'EarlyQuit',
     'MultipleIncorrectSubmissions',
     'CyclicStateTransitions'
 ]
-# IDs of allowed learner actions.
-ALLOWED_ACTION_IDS = [
+# Types of allowed learner actions.
+ALLOWED_ACTION_TYPES = [
     'ExplorationStart',
     'AnswerSubmit',
     'ExplorationQuit'
 ]
-
+# Mapping from issue type to issue keyname in the issue customization dict. This
+# mapping is useful to uniquely identify issues by the combination of their
+# issue type and the concerend state(s).
```

Comment:
typo: concerned

Also, change last part to: `... of their issue type and other type-specific information (such as the list of states involved).`

This gives more wiggle room in the future if we introduce keys for issues that are not related to states.

Reply:
Done.